entrypoint.sh

```diff
@@ -1,46 +1,75 @@
-#!/bin/sh -l
+#!/bin/bash
+set -e
 
-sourceDirectory=$1
-platform=$2
-platformVersion=$3
+checkArg() {
+    # Check if the input has content and is not just whitepsace or is empty
+    case $1 in
+    (*[![:blank:]]*) echo "$1";;
+    (*) echo ""
+    esac
+}
 
-echo
+getAbsolutePath(){
+    local path="$1"
+
+    if [ -z "$path" ]; then
+        echo "$PWD"
```

Comment:
nit: add additional echo that $path is empty

Reply:
We have that at the call site